scripts/ubuntu-bartender/qemu-provider

```diff
@@ -0,0 +1,178 @@
+# Beginning of "public" methods used by ubuntu-bartender
+
+function build-provider-assert-ready {
+  if ! command -v qemu-system-${QEMU_ARCH} &>/dev/null
+  then
+    echo "error: qemu-system-${QEMU_ARCH} was not found in PATH" >&2
+    exit 255
+  fi
+
+  if ! command -v cloud-localds &>/dev/null; then
+    echo "error: cloud-image-utils not found. You can install it with:"
+    echo "sudo apt install -y cloud-image-utils"
+    exit 255
+  fi
+
+  _set_disk_size
+
+  if [ -n "$QEMU_IMAGE" ]; then
```

Comment:
that `if` can be dropped. you are checking if the file exists in the next line anyway.